glossary.html

```diff
@@ -9,35 +9,37 @@
 	<nav>
 		<ul>
 			<li>
-			<a href="">Home</a>
+			<a href="index.html">Home</a>
 			</li>
 			<li>
-			<a href="">Links</a>
+			<a href="links.html">Links</a>
 			</li>
 			<li>
-			<a href="">Glossary</a>
+			<a href="glossary.html">Glossary</a>
 			</li>
 		</ul>
 	</nav>
 </header>
 
 <main>
 	<h1>Acronym Glossary</h1>
-		<h2>HTML Basic</h2>
-		<p>HTML defines the structure and layout of a Web document by using a variety of tags and attributes.</p>
-		<h2>CSS basics</h2>
-		<p>CSS describes how HTML elements are to be displayed on screen, paper, or in other media</p>
-		<h2>internet Basic</h2>
-		<p>The Internet is essentially a global network of computing resources. You can think of the Internet as a physical collection of routers and circuits as a set of shared resources</p>
-		<h2>HTTP</h2>
-		<p>HTTP means HyperText Transfer Protocol. HTTP is the underlying protocol used by the World Wide Web and this protocol defines how messages are formatted and transmitted, and what actions Web servers and browsers should take in response to various commands</p>
+	<dl>
+		<dt>HTML Basic</dt>
+		<dd>HTML defines the structure and layout of a Web document by using a variety of tags and attributes.</dd>
+		<dt>CSS basics</dt>
+		<dd>CSS describes how HTML elements are to be displayed on screen, paper, or in other media</dd>
+		<dt>internet Basic</dt>
+		<dd>The Internet is essentially a global network of computing resources. You can think of the Internet as a physical collection of routers and circuits as a set of shared resources</dd>
+		<dt>HTTP</dt>
+		<dd>HTTP means HyperText Transfer Protocol. HTTP is the underlying protocol used by the World Wide Web and this protocol defines how messages are formatted and transmitted, and what actions Web servers and browsers should take in response to various commands</dd>
+	</dl>
 		
 </main>
 <footer>
 	<hr>
-	<blockquote>"If you want to be happy, do not dwell in the past, do not worry about the future, focus on living fully in the present."</blockquote>
-	<blockquote> <span class="byline">― Roy T. Bennett, The Light in the Heart<span></blockquote>
-	<p>&copy; <a href="">Shivani sharma</a><p>
+	<blockquote>"If you want to be happy, do not dwell in the past, do not worry about the future, focus on living fully in the present."<br>
+	<em>&mdash; Roy T. Bennett, The Light in the Heart</em></blockquote>
+	<p>&copy; <a href="mailto:shivani@something.com?subject=Message%20from%20CSD120%20website">Shivani sharma</a><p>
```

Comment:
No need to create multipl blockquote elements.  Instead, just add a line break (`<br>`)
The author line was to be emphasized
You were asked to use an entity character to create the dash (ie, use `&mdash;`)